include/hobbes/net.H

```diff
@@ -324,7 +324,7 @@ _HNET_DEFINE_PRIMTYS(int32_t,  "int");
 _HNET_DEFINE_PRIMTYS(uint32_t, "int");
 _HNET_DEFINE_PRIMTYS(int64_t,  "long");
 _HNET_DEFINE_PRIMTYS(uint64_t, "long");
-_HNET_DEFINE_PRIMTYS(size_t,   "long");
+//_HNET_DEFINE_PRIMTYS(size_t,   "long");
```

Comment:
I just manually built my PR on mac, it works fine.  We need get clock_gettime fixed before we fully enable travis.